src/ui/layout/tree-item.vue

```diff
@@ -107,6 +111,18 @@ export default {
         }
 
         this.openmct.router.on('change:path', this.highlightIfNavigated);
+
+        this.getLocalStorageExpanded();
+        this.getLocalStorageNavigated();
+    },
+    beforeDestroy() {
+        /****
+            * calling this.setLocalStorageExpanded explicitly here because for whatever reason,
+            * the watcher on this.expanded is not triggering this.setLocalStorageExpanded(),
+            * even though Vue documentation states, "At this stage the instance is still fully functional."
+        *****/
+        this.expanded = false;
+        this.setLocalStorageExpanded();
```

Comment:
I'm not sure why setting this.expanded does not trigger the watcher so I'm calling this explicitly. If you know how to fix this that'd be better.